### PR TITLE
Change dependencies to kube 1.7

### PR DESCRIPTION
**Pull request by jboyd01**
updated glide.yaml to point to 1.7 releases of kube, apiserver, apimachinary, client-go
fixed code issues introduced by update, mostly around scheduler creation (simulator.go)

### CODE REVIEW

**Comment by aveshagarwal** [comment]
@jboyd01 the build is failing, can you check it?

```
go build -o hypercc github.com/kubernetes-incubator/cluster-capacity/cmd/hypercc
# github.com/kubernetes-incubator/cluster-capacity/vendor/k8s.io/apimachinery/pkg/util/strategicpatch
vendor/k8s.io/apimachinery/pkg/util/strategicpatch/patch.go:520: undefined: sort.SliceStable
make: *** [build] Error 2
```

**Comment by jboyd01** [comment]
looks to be an issue with travis using go 1.7.  It should be using 1.8.  I'll submit a new PR to move travis to 1.8

File: glide.yaml [hunk 1]
```diff
@@ -29,7 +32,7 @@ import:
   - pkg/types
   - pkg/conversion
 - package: k8s.io/kubernetes
-  version: v1.6.0
+  version: ebb8d6e0fadfc95f3d64ccecc36c8ed2ac9224ef # kube 1.7.0
```
Comment (aveshagarwal) [inline glide.yaml]: @jboyd01 just curious, do you know how ahead this SHA is from 1.7.0 release?  Or is it between 1.7.1 and 1.7.0 or in between 1.7.1 or 1.7.2 or even above 1.7.2?

Comment (jboyd01) [inline glide.yaml]: I followed the changes done on scheduler for moving to 1.7.  This SHA points to 

commit ebb8d6e0fadfc95f3d64ccecc36c8ed2ac9224ef
Author: Chao Xu <xuchao@silkubi.fra.corp.google.com>
Date:   Fri Jun 30 02:13:41 2017 +0200

    Kubernetes version v1.7.1-beta.0 file updates

diff --git a/api/openapi-spec/swagger.json b/api/openapi-spec/swagger.json
index 8cfb739..3f0dc11 100644
--- a/api/openapi-spec/swagger.json
+++ b/api/openapi-spec/swagger.json
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.7.0"
+   "version": "v1.7.1"
   },
   "paths": {
    "/api/": {
diff --git a/federation/apis/openapi-spec/swagger.json b/federation/apis/openapi-spec/swagger.json
index 6aca78d..daee7e0 100644
--- a/federation/apis/openapi-spec/swagger.json
+++ b/federation/apis/openapi-spec/swagger.json

**Comment by derekwaynecarr** [comment]
code LGTM, would like to verify it runs fine as a job on a cluster before merging.

**Pull request merged**